infrastructure/aws/ecs/backend-development.tmpl

```diff
@@ -0,0 +1,104 @@
+${jsonencode([
+  {
+    "dnsSearchDomains": null,
+    "logConfiguration": {
+      "logDriver": "awslogs",
+      "secretOptions": null,
+      "options": {
+        "awslogs-group": "/ecs/backend-development",
+        "awslogs-region": "${aws-region}",
+        "awslogs-stream-prefix": "ecs"
+      }
+    },
+    "entryPoint": [],
+    "portMappings": [
+      {
+        "hostPort": 8300,
+        "protocol": "tcp",
+        "containerPort": 8300
+      }
+    ],
+    "command": [],
```

Comment:
Myslim ze ti tu chybi nahozeni webserveru (zatim integrated dev server, pozdeji nahradime za neco produkcniho):
```
/app/manage.py runserver <server>:8300
```
Pokud tohle je to misto kde se ma spoustet i migrace, pak radeji:
```
/app/manage.py migrate && /app/manage.py runserver <server>:8300
```

V `docker-compose.yml` mame jako `<server>` hodnotu `backend`, tj. to same jmeno jako dany kontejner. Na AWS tam bude neco jako `127.0.0.1:8300`, `0.0.0.0:8300` nebo obecne `<ip>:8300` (nevim jaka je konvence v AWS ECS)

Podrobnosti viz https://docs.djangoproject.com/en/3.1/ref/django-admin/#runserver